c++/src/Reader.cc

```diff
@@ -1061,6 +1061,64 @@ namespace orc {
                                                   postscriptLength));
   }
 
+  std::map<uint32_t, BloomFilterIndex>
```

Comment:
Why not return a `std::vector<BloomFilterIndex>` ?

Reply:
Bloom filters are enabled for part of columns in the file. Therefore returning a vector containing all columns is not reasonable. Returning a map allows users to figure out which column has bloom filter index inside.